src/test/gcp/cloudfunctions.spec.ts

```diff
@@ -455,6 +455,30 @@ describe("cloudfunctions", () => {
         codebase: "my-codebase",
       });
     });
+
+    it("should derive hash from labels", () => {
+      expect(
+        cloudfunctions.endpointFromFunction({
+          ...HAVE_CLOUD_FUNCTION,
+          httpsTrigger: {},
+          labels: {
+            ...CLOUD_FUNCTION.labels,
+            [cloudfunctions.CODEBASE_LABEL]: "my-codebase",
```

Comment:
In general I wonder if we should delete the labels in endpointFromFunction so that tests don't have to worry about both the label and the property implementation of these features when creating fake data.

Reply:
I noticed this part. I was concerned about unintended side-effects, so I left the forced assignment as-is.